docs/.vuepress/config.js

```diff
@@ -99,6 +99,7 @@ module.exports = {
       {
         text: "Components",
         items: [
+          { text: "CSS Utilities", link: "/styles/utilities/" },
           { text: "Design Tokens", link: "/components/design-tokens/" },
           { text: "Component Variables", link: "/components/component-variables/" },
           { text: "Utilities", link: "/components/utilities/"},
```

Comment:
Should delete this entry here since the new one covers it

Reply:
I don't think the route was known when I initially added this 2 weeks ago. I moved mine to the one that was added 👍